bin/internal/update_engine_version.ps1

```diff
@@ -16,8 +19,23 @@ $ErrorActionPreference = "Stop"
 $progName = Split-Path -parent $MyInvocation.MyCommand.Definition
 $flutterRoot = (Get-Item $progName).parent.parent.FullName
 
+# Allow overriding the intended engine version via FLUTTER_PREBUILT_ENGINE_VERSION.
+#
+# This is for systems, such as Github Actions, where we know ahead of time the
+# base-ref we want to use (to download the engine binaries and avoid trying
+# to compute one below), or for the Dart HH bot, which wants to try the current
+# Flutter framework/engine with a different Dart SDK.
+#
+# This environment variable is EXPERIMENTAL. If you are not on the Flutter infra
+# or Dart infra teams, this code path might be removed at anytime and cease
+# functioning. Please file an issue if you have workflow needs.
+if (![string]::IsNullOrEmpty($env:FLUTTER_PREBUILT_ENGINE_VERSION)) {
+  $engineVersion = $env:FLUTTER_PREBUILT_ENGINE_VERSION
+  "[Unstable] Override: Setting engine SHA to $engineVersion" | Out-Stream -Stream Error
```

Comment:
`Write-Error` would have been fine if it wouldn't be for line 17 where `$ErrorActionPreference` is set to `Stop`: with it non-terminating `Write-Error` error reporting becomes terminating.

Reply:
Thanks!